src/billing_infos/swico/erro.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(Debug, thiserror::Error)]
```

Comment:
As mentioned elsewhere, the play on words in this filename, while amusing, goes against the principle of least surprise, and is probably more trouble than it is worth.